### PR TITLE
Remove broken xmlrpc ping

### LINKED ISSUE #9644: Remove xmlrpc ping service
Ghost has a built-in service for making xmlrpc pings when a post is published:

https://github.com/TryGhost/Ghost/blob/master/core/server/services/xmlrpc.js

These services are no longer working.

Therefore the service and related tests need to be removed.

### PR DESCRIPTION
**Pull request by Rahmon**
Remove xmlrpc service and related test,

Closes #9644.

- [ ] There's a clear use-case for this code change
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `npm test`)

### CODE REVIEW

**Comment by kirrg001** [comment]
Will review asap. Thanks for the PR 👍

File: core/server/services/xmlrpc.js [hunk 1]
```diff
@@ -1,101 +0,0 @@
...
-            url: 'rpc.pingomatic.com'
-        }
-    ];
-
-function ping(post) {
-    var pingXML,
-        title = post.title,
-        url = urlService.utils.urlFor('post', {post: post}, true);
-
-    if (post.page || config.isPrivacyDisabled('useRpcPing') || settingsCache.get('is_private')) {
```
Comment (kirrg001) [inline core/server/services/xmlrpc.js]: `useRpcPing` needs a double check. There is one value in the config file.

File: core/server/services/xmlrpc.js [hunk 2]
```diff
@@ -1,101 +0,0 @@
...
-            timeout: 2 * 1000
-        };
-
-        request(pingHost.url, options)
-            .catch(function (err) {
-                common.logging.error(new common.errors.GhostError({
-                    err: err,
-                    message: err.message,
-                    context: common.i18n.t('errors.services.ping.requestFailed.error', {service: 'xmlrpc'}),
-                    help: common.i18n.t('errors.services.ping.requestFailed.help', {url: 'http://docs.ghost.org'})
```
Comment (kirrg001) [inline core/server/services/xmlrpc.js]: Are these error message used somewhere else?
If no, we can remove them :)

**Review by kirrg001 — CHANGES_REQUESTED** [review]
2 tiny comments :)

File: core/server/services/xmlrpc.js (hunk 2 above)
Comment (kirrg001) [reply core/server/services/xmlrpc.js]: There is a translation file in `server/translations/`. You can look up the full keys e.g. `errors.services.ping.requestFailed.help` by doing a project search. If they are not used, you can remove them from the translation file.

Comment (Rahmon) [reply core/server/services/xmlrpc.js]: slack.js file use:
https://github.com/TryGhost/Ghost/blob/a3091a30123bd08e274fe0c5c4537366f719cb74/core/server/services/slack.js#L34

**Comment by kirrg001** [comment]
@Rahmon The PR needs a rebase :)

**Comment by kirrg001** [comment]
After merge and release we have to change our config page: https://docs.ghost.org/docs/config#section-rpc-ping

**Comment by Rahmon** [comment]
The [PRIVACY.md](https://github.com/TryGhost/Ghost/blob/master/PRIVACY.md#rpc-pings) file need to be update too. Do I wait merge/release or do it before?

**Comment by kirrg001** [comment]
Should be part of this PR 👍 
But i just played with the two services and one of it is still working 🤔 See comment on the issue.

**Comment by Rahmon** [comment]
@kirrg001 I've updated this PR. Just changed the method name from 'weblogUpdate.ping' to 'weblogUpdates.ping'. Something else is necessary?

**Comment by kirrg001** [comment]
@Rahmon Thanks! Did you test both RPC endpoints with this method? 🤔 
I thought one is definitely not working anymore and one is still working, but just with a different method name.

**Comment by Rahmon** [comment]
@kirrg001 how do you test RPC endpoints?

P.S.: I'm sorry I took such a long time to answer this.

**Pull request closed without merging**